packages/build-utils/src/clone-env.ts

```diff
@@ -16,14 +16,19 @@ export function cloneEnv(...envs: (Env | undefined)[]): Env {
       return obj;
     }
 
-    // the system path is called `Path` on Windows and Node.js will
-    // automatically return the system path when accessing `PATH`,
-    // however we lose this proxied value when we destructure and
-    // thus we must explicitly copy it
-    if (hasOwnProperty.call(env, 'PATH') || hasOwnProperty.call(env, 'Path')) {
-      obj.PATH = env.PATH;
+    // mixin the env first
+    obj = Object.assign(obj, env);
+
+    if (hasOwnProperty.call(env, 'Path')) {
+      // the system path is called `Path` on Windows and Node.js will
+      // automatically return the system path when accessing `PATH`,
+      // however we lose this proxied value when we destructure and
+      // thus we must explicitly copy it, but we must also remove the
+      // `Path` property since we can't have both a `PATH` and `Path`
+      obj.PATH = obj.Path;
+      delete obj.Path;
```

Comment:
**question (non-blocking):** Is it OK that `Path` will never be in the result?

Will all of the uses of the result of `cloneEnv` always work when `Path` is not there?

Reply:
Just to circle back, we can't have both `Path` and `PATH` since that's what causes this issue. A quick grep of the code yields no match where we're referencing the `PATH` as spelled `Path`. We should be good to go.

Reply:
I think this is is the purpose of the `cloneEnv()` function. After cloning, we will read `.PATH` and never `.Path`